pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -203,10 +237,57 @@ func (o *MustGatherOptions) completeImages() error {
 		}
 		o.Images = append(o.Images, image)
 	}
+	if o.AllImages {
+		// find all csvs and clusteroperators with the annotation "operators.openshift.io/must-gather-image"
+		pluginImages := make(map[string]struct{})
+		var err error
+
+		pluginImages, err = o.annotatedCSVs()
```

Comment:
`pluginImages` needs to be extended with `o.Images[0]` just in case the detected images have it duplicated.

Reply:
Done. PTAL if you find the approach OK. I am deleting the key from the map before we start appending:
```go
// delete the default image to avoid duplication in case an Operator had it in its annotation
delete(pluginImages, o.Images[0])
for i := range pluginImages {
    o.Images = append(o.Images, i)
}
```

Reply:
Deleting will not cover the case where the default image is not added through annotation.

Reply:
Marking resolved based on https://github.com/openshift/oc/pull/1633#pullrequestreview-1818164255.